hack/make/test-deb-install

```diff
@@ -12,7 +12,13 @@ fi
 
 test_deb_install(){
 	# test for each Dockerfile in contrib/builder
-	for dir in contrib/builder/deb/*/; do
+
+	builderDir="contrib/builder/deb"
+	pkgs=( $(find "${builderDir}/"*/ -type d) )
+	if [ ! -z "$DOCKER_BUILD_PKGS" ]; then
+		pkgs=( $(echo ${DOCKER_BUILD_PKGS[@]/#/$builderDir\/}) )
+	fi
+	for dir in "${pkgs[@]}"; do
 		local from="$(awk 'toupper($1) == "FROM" { print $2; exit }' "$dir/Dockerfile")"
```

Comment:
Nice!  Shouldn't this still include `[ -d "$dir" ] || { echo >&2 "skipping nonexistent $dir"; continue; }` as the first line in the `for` loop here though? (https://github.com/docker/docker/blob/0e4eda16913eda3d892b6025b00539b72535d274/hack/make/build-deb#L46)